ocs_ci/utility/kms.py

```diff
@@ -681,6 +700,52 @@ def validate_external_vault(self):
             logger.error("KMS not enabled on storage cluster")
             raise NotFoundError("KMS flag not found")
 
+    def create_csi_kms_resources(self):
+        """
+        Create vault specific OCP resources to be consumed by
+        storage class.
+
+        Currently we have to create 'ceph-csi-kms-token' and
+        'csi-kms-connection-details '
+
+        """
+        self.create_vault_csi_kms_connection_details()
+        self.create_vault_csi_kms_token()
+
+    def create_vault_csi_kms_token(self):
```

Comment:
Since this token is specific to each namespace in OCS where encrypted PVCs are created, I think it would be better to add a namespace argument as well

Reply:
done